www_src/lib/keyboard.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  simulateTabPress: function () {
+    var current = document.activeElement;
+    var elements = document.getElementsByTagName('input');
```

Comment:
In a mobile context all we really care about is `input` and `textarea`. Will add `textarea` for completeness.